studio/components/interfaces/Database/EnumeratedTypes/EnumeratedTypeValueRow.tsx

```diff
@@ -0,0 +1,54 @@
+import { Draggable, DraggableProvided } from 'react-beautiful-dnd'
+import { Button, IconMenu, IconTrash, Input } from 'ui'
+
+interface EnumeratedTypeValueRowProps {
+  index: number
+  isDisabled?: boolean
+  enumTypeValue: { id: string; value: string }
+  onUpdateValue: (id: string, value: string) => void
+  onRemoveValue: () => void
+}
+
+const EnumeratedTypeValueRow = ({
+  index,
+  isDisabled = false,
+  enumTypeValue,
+  onUpdateValue,
+  onRemoveValue,
+}: EnumeratedTypeValueRowProps) => {
+  return (
+    <Draggable draggableId={enumTypeValue.id} index={index} isDragDisabled={isDisabled}>
+      {(draggableProvided: DraggableProvided) => (
+        <div
+          ref={draggableProvided.innerRef}
+          {...draggableProvided.draggableProps}
+          className="flex items-center space-x-2"
+        >
+          <div
+            {...draggableProvided.dragHandleProps}
```

Comment:
could spread these props on the Icon component instead maybe.
the luide icons should work with that.

Reply:
hmm just tried but doesnt seem to work - react-beautiful-dnd's complaining that it can't find the drag handle.
i think this might be a react beautiful dnd thing though that it doesnt expect an SVG to be the drag handle. i can see that the drag handle props are rendering correctly in the html

Reply:
ok, makes sense!

Reply:
nevermind, carry on :D